js/index.js

```diff
@@ -57,299 +40,313 @@ for (year = firstYear; year <= lastYear; year++) {
 }
 yearSelector.lastChild.selected = true;
 
+function Map( selector, yearSelector ) {
```

Comment:
Hide whitespace changes in your diff settings to avoid seeing all the lines impacted by de-indenting the Map class moving it outside of the closure.